data/yam/autoyast/autoyast_scc_up.xml

```diff
@@ -8,9 +8,8 @@
       <boot_mbr>true</boot_mbr>
       <boot_root>true</boot_root>
       <generic_mbr>false</generic_mbr>
-      <timeout config:type="integer">5</timeout>
+      <timeout config:type="integer">-1</timeout>
     </global>
-    <loader_type>{{LOADER_TYPE}}</loader_type>
```

Comment:
Please reserve this line, it is for different loader type on different arches.

Reply:
Now I'm set the `LOADER_TYPE=grub2` in the test case's setting.

Reply:
@lemon-suse  now all VRs are passed, please take a look again, thanks!